.github/workflows/test.yml

```
@@ -0,0 +1,19 @@
+
+name: test
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 14]
```

```suggestion
        node: [12, 16]
```
Let's cover the full current range.

---

Or `12, 14, 16` ?

---

Personally I think start and end should be enough, but one might want to add 14 as well as it's a current LTS version...